server/model/json_test.go

```diff
@@ -149,29 +148,6 @@ func TestRunEncoding(t *testing.T) {
 	}
 }
 
-func TestOldAssertionSerialization(t *testing.T) {
```

Comment:
We are totally breaking retro-compatibility with this PR, so I'm taking away this test that was here to ensure retro-compatibility